aspnetcore/razor-pages/rp-mvc.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Razor Pages compared to ASP.NET Core MVC with controllers and views
+author: Rick-Anderson
+description: Contrast and compare Razor Pages to controller/view programming in ASP.NET Core
+monikerRange: '>= aspnetcore-2.0'
+ms.author: riande
+ms.date: 8/05/2018
+uid: razor-pages/rp-mvc
+---
+# Razor Pages compared to ASP.NET Core MVC
+
+By [Scott Sauber](https://twitter.com/scottsauber) and [Rick Anderson](https://twitter.com/RickAndMSFT)
+
+Razor Pages is a feature of the ASP.NET Core MVC framework. Razor Pages follows the MVVM pattern. Developers and teams familiar with ASP.NET MVC development:
```

Comment:
Can you link to an overview of MVVM?

Reply:
This sentence is an abrupt transition from starting to introduce RP to an apology saying you can still use MVC.  If this is aimed at MVC developers, I would save the apology for last and tell them what RP is and how it's different.

Reply:
It's aimed at everyone with special attention to MVC developers. I need to assure them right off the bat that MVC/controllers/views is not going away and that we're continuously investing in it. Read   #6146 to see what we're trying to address.